src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[];
+  onUpdateTodo: (id: number, parameter: Partial<Todo>) => void;
+  onDeleteTodo: (id: number) => void,
```

Comment:
Add a semicolon at the end of the line for consistency with the other prop types

Reply:
Done.